static/script.js

```diff
@@ -229,7 +229,7 @@ function downloadURL(url, name) {
 
 function alertIfFileContainsPassword() {
     passwords = []
-    let regex = /PASSWORD='.+'/i;
+    let regex = /<passphrase>.+<\/passphrase>|PASSWORD='.+'/i;
```

Comment:
A bit unrelated, but I just noticed that the ifcfg part also hardcodes the `'` character, however that isn't actually required and some people use `"` or even no quotes at all. So I think it should just be `PASSWORD=.+`

Reply:
Oh and some things I also forgot to mention previously (sorry) that it should also match:
In xml: `<password></password>`, `<client-key-passwd></client-key-passwd>`, `<key></key>` and `<modem-pin></modem-pin>`
In ifcfg: `WIRELESS_WPA_PASSWORD`, `WIRELESS_WPA_PSK`, `WIRELESS_KEY_[0123]` and `WIRELESS_CLIENT_KEY_PASSWORD`